src/formats/roam/utils.ts

```diff
@@ -11,7 +11,7 @@ const squareBracketCloseRe = /\]/g; // Regular expression to match "]"
 
 export function sanitizeFileNameKeepPath(name: string) {
 	return name
-		.replace(illegalReNoDir, '')
+		.replace(illegalReNoDir, '_')
```

Comment:
I'm not sure why this is changing. Can you explain why this is needed?

Reply:
It's not needed (and can remove it), I thought it was good to have a replacement rather than just deleting the characters. I'll take it out, and might add a feature later to optionally allow choosing a replacement character